docs/userdocs/k8s/deploy-submarine.md

```diff
@@ -0,0 +1,70 @@
+
+
+# Deploy Submarine On K8s
+
+## Deploy Submarine using Helm Chart (Recommended)
+
+### Create images
+submarine server
+```bash
+./dev-support/docker-images/submarine/build.sh
+```
+
+submarine database
+```bash
+./dev-support/docker-images/database/build.sh
+```
```

Comment:
If the user select to build docker image by manual, he/her should to modify the `values.yaml` and specify the correctly image in helm charts.